vars/RedHat_8.yml

```diff
@@ -58,6 +58,46 @@ __sap_hana_preconfigure_req_repos_RedHat_8_4_ppc64le:
   - "rhel-8-for-ppc64le-appstream-e4s-rpms"
   - "rhel-8-for-ppc64le-sap-solutions-e4s-rpms"
 
+__sap_hana_preconfigure_req_repos_RedHat_8_5_x86_64:
+  - "rhel-8-for-x86_64-baseos-rpms"
+  - "rhel-8-for-x86_64-appstream-rpms"
+  - "rhel-8-for-x86_64-sap-solutions-rpms"
+
+__sap_hana_preconfigure_req_repos_RedHat_8_5_ppc64le:
+  - "rhel-8-for-ppc64le-baseos-rpms"
+  - "rhel-8-for-ppc64le-appstream-rpms"
+  - "rhel-8-for-ppc64le-sap-solutions-rpms"
+
+__sap_hana_preconfigure_req_repos_RedHat_8_6_x86_64:
+  - "rhel-8-for-x86_64-baseos-e4s-rpms"
+  - "rhel-8-for-x86_64-appstream-e4s-rpms"
+  - "rhel-8-for-x86_64-sap-solutions-e4s-rpms"
+
+__sap_hana_preconfigure_req_repos_RedHat_8_6_ppc64le:
+  - "rhel-8-for-ppc64le-baseos-e4s-rpms"
+  - "rhel-8-for-ppc64le-appstream-e4s-rpms"
+  - "rhel-8-for-ppc64le-sap-solutions-e4s-rpms"
+
+__sap_hana_preconfigure_req_repos_RedHat_8_7_x86_64:
+  - "rhel-8-for-x86_64-baseos-rpms"
+  - "rhel-8-for-x86_64-appstream-rpms"
+  - "rhel-8-for-x86_64-sap-solutions-rpms"
+
+__sap_hana_preconfigure_req_repos_RedHat_8_7_ppc64le:
+  - "rhel-8-for-ppc64le-baseos-rpms"
+  - "rhel-8-for-ppc64le-appstream-rpms"
+  - "rhel-8-for-ppc64le-sap-solutions-rpms"
+
+__sap_hana_preconfigure_req_repos_RedHat_8_8_x86_64:
+  - "rhel-8-for-x86_64-baseos-e4s-rpms"
```

Comment:
repos for RHEL 8,6 and RHEL 8.8 have an `e4s` part in them, while repos for RHEL 8.5 and 8.7 do not. Is that expected?

Reply:
Yes, we expect eus and e4s repos for RHEL 8.(even) and none of these for RHEL 8.(uneven). See also the last sentence in chapter https://access.redhat.com/support/policy/updates/errata#Update_Services_for_SAP_Solutions . Although HANA will likely not be validated for RHEL 8.(uneven), I want to make the roles can run there as well, at least just for test purposes.

Reply:
OK, thank you for the clarification